molecule/default/tests/test_default.py

```diff
@@ -1,14 +1,51 @@
 import os
-
+import pytest
 import testinfra.utils.ansible_runner
 
+
+"""ASC-240: Verify the requested glance images were uploaded
+
+RPC 10+ manual test 10
+"""
+
 testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
-    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('compute-infra_hosts')[:1]
+
+# attach the utility container:
+attach_utility_container = "lxc-attach -n `lxc-ls -1 | grep utility | head -n 1` -- bash -c "
+
 
+@pytest.mark.jira('asc-240')
+@pytest.mark.xfail(reason='expected to fail until service setup is successfully completed')
+def test_verify_glance_image(host):
+    """Verify the glance images created by:
+    https://github.com/openstack/openstack-ansible-ops/blob/master/multi-node-aio/playbooks/vars/openstack-service-config.yml
+    """
+    cmd = attach_utility_container + "'. /root/openrc ; openstack image list'"
+    output = host.run(cmd)
+    assert ("Ubuntu 14.04 LTS" in output.stdout)
+    assert ("Ubuntu 16.04" in output.stdout)
+    assert ("Fedora 27" in output.stdout)
+    assert ("CentOS 7" in output.stdout)
+    assert ("OpenSuse Leap 42.3" in output.stdout)
+    assert ("Debian 9 Latest" in output.stdout)
+    assert ("Debian TESTING" in output.stdout)
+    assert ("Cirros-0.3.5" in output.stdout)
 
-def test_hosts_file(host):
-    f = host.file('/etc/hosts')
 
-    assert f.exists
-    assert f.user == 'root'
-    assert f.group == 'root'
+@pytest.mark.jira('asc-240')
+@pytest.mark.xfail(reason='expected to fail until service setup is successfully completed')
+def test_verify_subnet_list(host):
```

Comment:
This test name is still incorrect. The code below tests the glance `flavors` not subnets.

I don't understand why this is expected to fail. What service setup is necessary to run `openstack` commands on the utility container?